aroquev/src/main/webapp/script.js

```diff
@@ -221,15 +221,11 @@ function getServletComments() {
 function verifyNumberCommentsInput() {
   // Default value
   let maxComments = 5;
-
-  let maxCommentsInput = document.getElementById("maxComments").value;
-  const commentsErrorContainer = document.getElementById('js-comments-input-error-container');
+  let maxCommentsContainer = document.getElementById("comment-number-input");
+  maxCommentsContainer.reportValidity();
```

Comment:
Maybe you want to prevent the request to get comments if the number is invalid. As far as I can tell, if a user types in "20", they will see an error and then 5 comments will be fetched anyway. This might be non-obvious behavior to the user.

Reply:
Gotcha, will implement that